.github/workflows/qmkpid.yml

```
@@ -0,0 +1,57 @@
+name: Generate QMKPID
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  generate_pid:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out
+      uses: actions/checkout@v1
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.5'
+
+    - name: Cache pip
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip # This path is specific to Ubuntu
+        # Look to see if there is a cache hit for the corresponding requirements file
+        key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+          ${{ runner.os }}-
+
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+
+    - name: Set up git config
+      run: |
+        git config --local user.email "action@github.com"
```

We should commit as bot here
```suggestion
        git config --global user.name "QMK Bot"
        git config --global user.email "hello@qmk.fm"
```

---

Done :)